textproc/json2tsv/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                json2tsv
+version             1.0
+revision            1
```

Comment:
`revision` for a completely brand new port, or a bump to a new version, should always start at 0.